Makefile

```diff
@@ -15,9 +16,13 @@ test:
 
 lint:
 	${LINT} run --build-tags=all
+	DIFF=$$(${FORMATTER} -d core); if [[ -n "$$DIFF" ]]; then printf "\n$$DIFF" && exit 1; fi
```

Comment:
Interesting, I'd never seen the use of $$ like this before (i.e. to capture the output of a command).  I was aware of `$(<command>)`, but not `$$(<command>)`.  In fact, I can't really even find where that is documented in the bash man page - lol.  I know it works ok on Linux, though, because I pulled down your branch and tried it :)

Reply:
Ah, now I think I know what's going on...  I think the extra $'s are needed because these commands are in a Makefile, rather than a bash script, amirite?  Do I win a prize? :joy:

Reply:
Haha, of course! This is yours: 🥇
 
I wanted to avoid embedding a "complicated" shell script into the Makefile or having a separate script, but overall I think this oneliner is not that bad.

Reply:
agreed, this is fine.  I just missed the fact that it was in a make file.